python/example01.py

```diff
@@ -0,0 +1,149 @@
+import pycurl
+import os
+import sys
+import json
+import io
+
+AK_SIZE = 20
+SK_SIZE = 40
+
+class Tag:
+    def __init__(self, resources_id, key, value, resources_type):
+        self.resources_id = resources_id
+        self.key = key
+        self.value = value
+        self.resources_type = resources_type
+
+def setup_curl(url, json_data, ak_sk):
+    curl = pycurl.Curl()
+    headers = ["Content-Type: application/json"]
+    curl.setopt(pycurl.HTTPHEADER, headers)
+    curl.setopt(pycurl.URL, url)
+    curl.setopt(pycurl.POSTFIELDS, json_data)
+    curl.setopt(pycurl.AWS_SIGV4, 'osc')
+    curl.setopt(pycurl.USERPWD, ak_sk)
+
+    response_data = []
```

Comment:
I guess this could be a string, on which you append the result in each `write_data`